generator/internal/codegen_utils_test.cc

```diff
@@ -29,10 +29,20 @@ TEST(LocalInclude, Success) {
             LocalInclude("google/cloud/status.h"));
 }
 
+TEST(LocalInclude, Empty) {
+  std::string empty;
```

Comment:
Nit: Whenever I have a variable whose name asserts something about its value, I tend to make it `const`.